src/dom/scroll.js

```diff
@@ -45,29 +45,33 @@ export function scrollViewportToShowTarget( { target, viewportOffset = 0 } ) {
 			firstAncestorToScroll = getParentElement( currentFrame );
 		}
 
-		// Scroll the target's ancestors first. Once done, scrolling the viewport is easy.
-		scrollAncestorsToShowRect( firstAncestorToScroll, () => {
-			// Note: If the target does not belong to the current window **directly**,
-			// i.e. it resides in an iframe belonging to the window, obtain the target's rect
-			// in the coordinates of the current window. By default, a Rect returns geometry
-			// relative to the current window's viewport. To make it work in a parent window,
-			// it must be shifted.
-			return getRectRelativeToWindow( target, currentWindow );
-		} );
-
-		// Obtain the rect of the target after it has been scrolled within its ancestors.
-		// It's time to scroll the viewport.
-		const targetRect = getRectRelativeToWindow( target, currentWindow );
-
-		scrollWindowToShowRect( currentWindow, targetRect, viewportOffset );
-
-		if ( currentWindow.parent != currentWindow ) {
-			// Keep the reference to the <iframe> element the "previous current window" was
-			// rendered within. It will be useful to re–calculate the rect of the target
-			// in the parent window's relative geometry. The target's rect must be shifted
-			// by it's iframe's position.
-			currentFrame = currentWindow.frameElement;
-			currentWindow = currentWindow.parent;
+		if ( firstAncestorToScroll ) {
```

Comment:
What does this condition actually do? In which cases there's no `firstAncestorToScroll`? 

As I [wrote](https://github.com/ckeditor/ckeditor5/issues/930#issuecomment-377495214) I tested the editor inside an iframe (not JSFiddle), and I couldn't reproduce the issue. What makes your use–case different?

This condition disables scrolling (ancestors, window) completely, which is wrong IMO. I see no good reason why it should be disabled; the bugs should be fixed instead.

Reply:
When I start debugging ckeditor inside my application, I get the following result when I press ```enter```:

* ```currentFrame``` equal null;
* ```currentWindow``` different of ```targetWindow```;
* ```getParentElement``` is called with a null param;
* console output error when the ```getParentElement``` try to return ***parentNode*** of null.

The changes I have made just make sure that ```elementOrRange``` isn't null.